2017/Day 4 - 2.py

```diff
@@ -0,0 +1,15 @@
+import itertools
+
+valid_passphrases_count = 0
+passphrases = []
+
+with open('Day 4 - input', 'r') as f:
+    for line in f:
+        passphrases.append(list(line.split()))
+    for passphrase_list in passphrases:
+        passphrase = list(map(lambda x: ('').join(sorted(list(x))), passphrase_list))
```

Comment:
It is preferable to use a `list comprehension` than calling `list` on `map`.
Also, sorted works fine on strings, no need to convert it to a list. Though the return object is a list and needs to be joined